.rubocop.yml

```diff
@@ -13,9 +14,12 @@ AllCops:
 # Metrics/LineLength:
 #   Max: 120
 
+
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
+  Exclude:
+    - 'Gemfile'  # until engine_cart stanza is violation free
```

Comment:
currently, engine_cart creates a "stanza" of chars that it puts at the end of your Gemfile.  That stanza has rubocop errors per default rubocop settings.  If you correct those rubocop errors in your Gemfile, then the stanza doesn't perfectly match any more.  In theory, engine_cart doesn't care, but I have a PR in to engine_cart  to fix the stanza.